cmd/service.go

```diff
@@ -28,45 +32,67 @@ var serviceCmd = &cobra.Command{
 }
 
 var serviceCreateCmd = &cobra.Command{
-	Use:   "create <service_type> [service_name]",
+	Use:   "create <service_type> --plan <plan_name> [service_name]",
 	Short: "Create a new service",
-	Long: `Create a new service from service catalog to deploy on OpenShift.
+	Long: `Create a new service from service catalog using the plan defined and deploy it on OpenShift.
 
-If service name is not provided, service type value will be used.
+If service name is not provided, service type value will be used. The plan to be used must be passed along the service type
+using this convention <service_type>/<plan>. The parameters to configure the service are passed as a list of key=value pairs.
+The list of the parameters and their type is defined according to the plan selected.
 
 A full list of service types that can be deployed are available using: 'odo catalog list services'`,
-	Example: `  # Create new mysql-persistent service from service catalog.
-  odo service create mysql-persistent
+	Example: `  # Create new postgresql service from service catalog using dev plan and name my-postgresql-db.
+  odo service create dh-postgresql-apb/dev my-postgresql-db --plan dev -p postgresql_user=luke -p postgresql_password=secret
```

Comment:
`odo service create dh-postgresql-apb`
since we have flag for `plan`, we can remove `dh-postgresql-apb/dev`

Reply:
@surajnarwade It was decided [here](https://github.com/redhat-developer/odo/issues/652#issuecomment-429813904) that we should drop the component to component link.
If you feel this should not happen, feel free to raise a red flag :)

Reply:
Good catch, will fix